src/main/java/gale/Event.java

```diff
@@ -0,0 +1,50 @@
+package gale;
```

Comment:
You could consider further separating your packages into `gale.tasks` and other items!